carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/android/AndroidService.java

```diff
@@ -1358,8 +1360,7 @@ private void openDateTimeSettingsSetupWizard(boolean turnOffAuto, TimeFormat tim
         }
 
         setSystemTime(timeFormat);
-
-        openApp("com.android.settings/.DateTimeSettingsSetupWizard");
+        openApp("com.android.settings/.Settings\\$DateTimeSettingsActivity");
```

Comment:
Please check on Android 7 and older. Looks like there should be checking for android version

Reply:
The ".Settings\\$DateTimeSettingsActivity" is actually working on all (6/7/8), it's backwards compatible. That was the purpose of such change, since ".DateTimeSettingsSetupWizard" was working only on 6.x and 7.x.

Reply:
Ok. Good to know.